### ISSUE TITLE
Make cache clearing instructions its own page 

### ISSUE THREAD

**Issue by scottcwilson** [zencart/documentation#408]
Take the content from 

https://docs.zen-cart.com/user/new_user_topics/change_header_logo/#image-caching

and put it in a separate article (cross link from the old page). 

For each of the steps, take a screenshot to show how it looks.  Remember these are *inexperienced people* who will be reading this article, so each step needs to be broken down. 

Once this is done there are likely other cross linking opportunities, but let's do that as a separate PR.

@wiztechinc Can you take this?

**Comment by wiztechinc** [comment]
Yes, but...of course I have changes for that. I have other ways of dealing with that issue and other observations. I’ll look deeper after I finish the other.

**Comment by scottcwilson** [comment]
Closed by #413.
